make_benchmark/zenodo_upload.py

```diff
@@ -0,0 +1,141 @@
+"""
+Zenodo Uploader: Uploads the datasets to the Geo-Bench community on Zenodo.
+
+By: Alexandre Drouin (ServiceNow Research)
+Date: June 2, 2023
+
+"""
+import json
+import os
+import requests
+
+from path import Path
+
+
+ACCESS_TOKEN = os.environ["ZENODO_TOKEN"]
```

Comment:
Perhaps put a quick docstring at the top on how to use the uploader. Or on the make_benchmark/README.md. This readme is currently non-existant. But I intend to populate it to keep track of all steps for building the benchmark.